app/src/main/res/drawable/target_arrow.xml

```diff
@@ -1,15 +1,7 @@
-<vector xmlns:android="http://schemas.android.com/apk/res/android"
-    android:width="24dp"
-    android:height="24dp"
-    android:viewportWidth="415.88"
-    android:viewportHeight="415.88">
-  <path
-      android:pathData="M179.64,189.57c2.45,0 4.87,0.19 7.22,0.56l37,-37c-13.19,-7.05 -28.25,-11.05 -44.22,-11.05c-51.92,0 -94.16,42.24 -94.16,94.16c0,51.92 42.24,94.16 94.16,94.16s94.16,-42.24 94.16,-94.16c0,-15.97 -4,-31.03 -11.05,-44.22l-37,37c0.37,2.35 0.56,4.77 0.56,7.22c0,25.74 -20.94,46.67 -46.67,46.67c-25.74,0 -46.67,-20.94 -46.67,-46.67S153.9,189.57 179.64,189.57z"
-      android:fillColor="#000000"/>
-  <path
-      android:pathData="M290.45,164.32c13.49,20.71 21.34,45.42 21.34,71.92c0,72.87 -59.28,132.15 -132.15,132.15c-72.87,0 -132.15,-59.28 -132.15,-132.15s59.28,-132.15 132.15,-132.15c26.51,0 51.21,7.85 71.92,21.34l34.1,-34.1c-29.74,-21.82 -66.4,-34.72 -106.03,-34.72c-99.06,0 -179.64,80.59 -179.64,179.64c0,99.05 80.59,179.64 179.64,179.64c99.05,0 179.64,-80.59 179.64,-179.64c0,-39.63 -12.9,-76.29 -34.72,-106.03L290.45,164.32z"
-      android:fillColor="#000000"/>
-  <path
-      android:pathData="M415.42,56.64c-1.12,-3.54 -4.12,-6.16 -7.78,-6.79l-35.45,-6.16l-6.16,-35.45c-0.64,-3.66 -3.26,-6.66 -6.79,-7.77c-3.54,-1.12 -7.4,-0.18 -10.03,2.45l-27.41,27.41c-1.86,1.86 -2.91,4.39 -2.91,7.03l0,40.9l-148.1,148.1c-5.18,5.18 -5.18,13.57 0,18.74c5.18,5.18 13.57,5.18 18.74,0L337.63,96.99h40.9c2.63,0 5.16,-1.05 7.03,-2.91l27.41,-27.41C415.59,64.04 416.54,60.18 415.42,56.64z"
-      android:fillColor="#000000"/>
+<vector android:autoMirrored="true" android:height="24dp"
+    android:viewportHeight="24" android:viewportWidth="24"
+    android:width="24dp" xmlns:android="http://schemas.android.com/apk/res/android">
+    <path android:fillColor="#FF09244B" android:fillType="nonZero"
```

Comment:
I think the fill color should depend on the theme, no? `android?<colorSomething>` if I do remember correctly the syntax

Reply:
This is an icon that I downloaded from the Internet. I would agree with you, but Jetpack Compose has no problem setting the tint anyway, so it doesn't matter in this case.

Reply:
Oh, perfect then!